src/Microsoft.Health.Fhir.Core/Models/CodingInfo.cs

```diff
@@ -21,5 +22,10 @@ public CodingInfo(string system, string code)
         public string System { get; set; }
 
         public string Code { get; set; }
+
```

Comment:
Do we still need this? `Coding` seems to be in Hl7.Fhir.Support NuGet package now so that's version agnostic right?

Reply:
We don't need this class anymore, I think I'll defer this to a different PR